fractions/jaeger/src/main/resources/modules/com/uber/jaeger/main/module.xml

```diff
@@ -0,0 +1,18 @@
+<module xmlns="urn:jboss:module:1.3" name="com.uber.jaeger">
+  <resources>
+    <artifact name="com.uber.jaeger:jaeger-core:${version.jaeger}"/>
+    <artifact name="com.uber.jaeger:jaeger-tracerresolver:${version.jaeger}"/>
+    <artifact name="com.uber.jaeger:jaeger-thrift:${version.jaeger}"/>
+    <artifact name="org.apache.thrift:libthrift:0.9.2"/>
```

Comment:
Alright, makes sense. I kept the versions fixed here, because this is something I did want to get feedback before getting it merged :)